src/provisioner/const.py

```diff
@@ -25,3 +25,7 @@
 # 'UPGRADE_MODE_VAL' constant for default upgrade mode.
 UPGRADE_MODE_VAL = ""
 REQUIRED_EXTERNAL_SW = ['kafka', 'consul']
+DELTA_PATH = "yaml:///etc/cortx/delta.conf"
+DELTA_IDX = 'delta_index'
```

Comment:
DELTA_IDX need not be part of const. Its a temp variable.

Reply:
removed from const.py